app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) == 3:
+        source_file = parts[1]
+        destination_file = parts[2]
+        if parts[0] == "cp" and source_file != destination_file:
```

Comment:
Here you should use unpacking, it will make your code more readable.
```suggestion
        cmd, source_file, destination_file = parts
        if cmd == "cp" and source_file != destination_file:
```